tools/check_format.py

```diff
@@ -558,8 +565,15 @@ def fixBuildPath(file_path):
     sys.stdout.write(fixBuildLine(line, file_path))
 
   error_messages = []
+
+  run_build_fixer = True
```

Comment:
This doesn't seem like it's used anymore?

Reply:
Good catch, missed that.